tests/apps/lit/test_lit_api.py

```diff
@@ -99,6 +99,25 @@ def test_reference_year_histogram(self, db_keys):
         resp = c.get(url).json()
         assert resp["data"][0]["type"] == "histogram"
 
+    def test_tag_heatmap(self, rewrite_data_files: str, db_keys):
```

Comment:
like above, can a helper function be written to test `test_references_download` and this function?